.gitmodules

```diff
@@ -21,10 +21,6 @@
 [submodule "main/external/nrefactory"]
 	path = main/external/nrefactory
 	url = https://github.com/dotdevelop/NRefactory.git
-[submodule "main/external/nuget-binary"]
-	path = main/external/nuget-binary
-	url = https://github.com/dotdevelop/nuget-binary.git
-	branch = master
```

Comment:
this could be now:
	branch = dotdevelop

Reply:
After our discussion in [dotdevelop/nuget-binaries](https://github.com/dotdevelop/nuget-binary/pull/1), I'll keep the deletions in .gitmodules above, but update [dotdevelop/nuget-binaries](https://github.com/dotdevelop/nuget-binary) if there are any future changes to the NuGet dlls